microservice-transaction-sample/README.md

```diff
@@ -418,39 +427,55 @@ for (ItemOrder itemOrder : request.getItemOrderList()) {
 
 Then, the Order Service calls the `payment` gRPC endpoint of the Customer Service along with the transaction ID. For reference, see [`OrderService.java`](order-service/src/main/java/sample/order/OrderService.java).
```

Comment:
@komamitsu @josh-wong Don't we need to specify the line number of the source code file in this URL?

Reply:
It was changed in https://github.com/scalar-labs/scalardb-samples/pull/50 and I think it was good since users can still search for the code on the class file and we can reduce the maintenance effort to always update the line numbers.

Reply:
Okay, understood. Let's keep it as it is.